descwl_shear_sims/shifts.py

```diff
@@ -218,6 +236,11 @@ def get_random_shifts(*, rng, dim, buff, size):
     """
 
     halfwidth = (dim - 2*buff)/2.0
+    if halfwidth < MIN_R_SHIFT:
```

Comment:
Is the idea that the user might use a buffer that is too  large?

Reply:
Maybe we should actually raise an exception if `dim - 2*buff < 0`

Reply:
Done!!